models/user.js

```diff
@@ -31,6 +35,51 @@
     });
   };
 
+  // Instance methods
+
+  UserSchema.methods.verifyUserPassword = function(password) {
+    if (password === undefined || password === null || password.length === 0) {
+      throw Error('No password to set.');
+    }
+    if (password.length > MAX_PASSWORD_LENGTH) {
+      throw Error('Password is too long.');
+    }
+  };
+
+  /*
+   * Set user's password.
+   */
+  UserSchema.methods.setPassword = function(rawPassword, callback) {
+    this.verifyUserPassword(rawPassword);
+    var password = authUtils.makePassword(rawPassword);
+    this.model('User').findByIdAndUpdate(this._id, {$set: {password: password}}, function(err, user) {
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'user' is defined but never used.